bin/www

```diff
@@ -13,5 +13,5 @@ app.set 'port', process.env.PORT
 
 console.log "Environment: #{app.get('env')}"
 
-server = app.listen app.get('port'), ->
+server = app.listen require('systemd-socket')() || app.get('port'), ->
```

Comment:
I'm not sure exactly what this does. Does this make the systemd-socket dependency optional? If so, perhaps the systemd-socket dependency in package.json should be moved to [optionalDependencies](https://docs.npmjs.com/files/package.json#optionaldependencies) and a note added to the readme about this.

Can the package still be run on a port after doing this? Does the user need further instruction now on how to configure for a port or a socket?